packages/opc-nav/src/opc-nav-search.scss

```diff
@@ -0,0 +1,65 @@
+@import '@one-platform/opc-styles';
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+ul,
+button,
+input {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  outline: 0;
+  font-weight: inherit;
+  vertical-align: baseline;
+  background: none;
+}
```

Comment:
@gisumit aren't all these changes already available in opc-styles, @akhilmhdh can you check?

Reply:
The margin & paddings are reset in the opc-styles package.

Reply:
margin padding yes but not some of them